app/assets/javascripts/discourse/app/instance-initializers/viewport-setup.js

```diff
@@ -0,0 +1,29 @@
+let done = false;
+
+export default {
+  initialize(container) {
+    if (done) {
+      return;
+    }
+    done = true;
+
+    const caps = container.lookup("service:capabilities");
+    const viewport = document.querySelector("meta[name=viewport]");
+    if (!viewport) {
+      return;
+    }
+
+    let value = viewport.getAttribute("content");
+    if (caps.isIOS()) {
+      // In iOS Safari, setting user-scalable=no doesn't actually prevent the user from zooming in.
+      // But, it does prevent the annoying 'auto zoom' when focussing input fields with small font-sizes.
+      value += ", user-scalable=no";
+    }
+
+    if (caps.isChrome) {
```

Comment:
This should apply to Firefox as well. In fact, maybe we extend it to all non-Safari browsers, just in case?

Reply:
Good call - done